README.md

```diff
@@ -33,6 +33,16 @@ demos of each.
   * [Relevant Material](#relevant-material)
 <!--te-->
 
+### Run Demo in a Google Cloud Shell
+
+Click the button below to run the demo in a [Google Cloud Shell][10].
```

Comment:
`[Google Cloud Shell][10] needs to be changed to [Google Cloud Shell](https://cloud.google.com/shell/docs/)`

Reply:
please test this